TFT/src/User/API/Vfs/vfs.c

```diff
@@ -132,63 +132,88 @@ bool IsRootDir(void)
 }
 
 // check if filename provides a supported filename extension
-char * isSupportedFile(char * filename)
```

Comment:
not my usual coding style but in those file it was used the style providing a space for the type (e.g. char * filename instead of char *filename). I would restore (maintain the same style as it was before) the previous style (so char * isSupportedFile(...) instead of char *isSupportedFile(...)

Reply:
Done.